molgenis-app-manager/src/main/java/org/molgenis/app/manager/controller/AppManagerController.java

```diff
@@ -71,6 +74,11 @@ public void uploadApp(@RequestParam("file") MultipartFile multipartFile) throws
 		InputStream fileInputStream = multipartFile.getInputStream();
 		String filename = multipartFile.getOriginalFilename();
 		String formFieldName = multipartFile.getName();
-		appManagerService.uploadApp(fileInputStream, filename, formFieldName);
+		String tempDir = appManagerService.uploadApp(fileInputStream, filename, formFieldName);
```

Comment:
several of these calls throw a io exception, i would think we would need to handle this